### DISCUSSION TITLE
Candle exe not being found

### DISCUSSION THREAD

**Discussion by Jorran** [wixtoolset/issues#7691] [Questions] [answered]
after install with the dotnet tool install command and everything going fine version check as well, the command candle.exe can't be found. Any reason this might happen? Because i did the exact same thing to start the development as i did on another computer where it works fine.

**Comment by robmen** [comment, marked as answer]
You might find https://wixtoolset.org/docs/fourthree/ useful.

**Reply by DavidWindham** [reply]
~There's no mention of how to install candle, nor the deprecation of it in the documentation. For everyone else who wants an answer rather than a dismissive "read the docs" reply, here's an actual answer.~
Edit: I was wrong on this bit, I think the documentation could be clearer, especially for those coming to find a quick answer, but it is there. I was unnecessarily shirty.

Candle was deprecated when Wix moved from V3 to V4. This has had the effect of causing some headaches if you're using one of the external tools that's built upon wix, e.g. cargo-wix.

Cargo-wix only supports V3, so if you wish to use it, or similar tooling, then you'll need to grab the latest V3 release which is v3.14.1 at time of writing, available here: https://github.com/wixtoolset/wix3/releases

Install that and you'll be good to go, but ensure you're reading from the V3 docs rather than the later V4 or V5 versions.

**Reply by robmen** [reply]
The change is mentioned in the second bullet of the [WiX v4 for WiX v3 users documentation](https://wixtoolset.org/docs/fourthree/). Seriously, people complain that there isn't enough documentation, and when there is, now you complain when I don't copy and paste the documentation into the question for you.

Note: those other tools need to be updated. I don't know why you are complaining here that they are out of date. We don't work on those tools.

**Reply by DavidWindham** [reply]
@robmen Alright I was wrong there, I had seen that page but I only skim-read it. I looked for "deprecated" or similar explicit language, but that's also on me for not reading as closely. I've edited my original message to strike-through the initial part.
